recipes/deploy-admin.rb

```diff
@@ -116,7 +113,6 @@
         s3_distribution_bucket_name: s3_distribution_bucket_name,
         capture_agent_monitor_url: capture_agent_monitor_url,
         live_streaming_url: live_streaming_url,
-        using_ssl_for_engage: using_ssl_for_engage,
```

Comment:
is this a residual change from earlier refactoring? I can't seem to trace the relevance.